tiny_obj_loader.h

```diff
@@ -1939,6 +1946,33 @@ bool LoadObj(attrib_t *attrib, std::vector<shape_t> *shapes,
       tags.push_back(tag);
     }
 
+	if (token[0] == 's' && IS_SPACE(token[1])) {
+	  // smoothing group id
+	  token += 2;
+	  int smGroupId = parseInt(&token);
```

Comment:
The .obj specification says that, "s" can be given the value of "off" instead of 0 to tell that there is no smoothing group.
I shall check to see if I have to add more code to handle that case.